web/src/cypress/support/alert.ts

```diff
@@ -172,6 +177,36 @@ function createAlert(a?: AlertOptions): Cypress.Chainable<Alert> {
     .then(res => res.createAlert)
 }
 
+function createManyAlerts(
+  count: number,
+  alertOptions?: AlertOptions,
+): Cypress.Chainable {
+  if (!alertOptions?.serviceID) {
+    return cy
+      .createService(alertOptions?.service)
+      .then(res =>
+        createManyAlerts(count, { ...alertOptions, serviceID: res.id }),
+      )
+  }
+
+  // build query
+  let query =
+    'insert into alerts (service_id, summary, details, dedup_key) values '
+  let rows: Array<string> = []
+  for (let i = 0; i < count; i++) {
+    const summary = alertOptions.summary || c.word()
+    const details = alertOptions.details || c.sentence()
+    const dedupKey = 'manual:1:createManyAlerts_' + i
```

Comment:
Declare a variable outside this function, and increment it and use that instead of the `for` var. Otherwise if you call this twice on the same DB you'll get identical dedup keys.